src/scaffoldmaker/meshtypes/meshtype_3d_colon1.py

```diff
@@ -441,47 +441,41 @@ def generateBaseMesh(cls, region, options):
         elementsAlongInDistal = elementsCountAlong - elementsAlongInProximal - elementsAlongInTransverse
         elementsCountAlongGroups = [elementsAlongInProximal, elementsAlongInTransverse, elementsAlongInDistal]
 
-        annotationGroups = []
         colonGroup = AnnotationGroup(region, get_colon_term("colon"))
-        annotationGroupsAlong = [colonGroup]
 
         if tcCount == 1:
             proximalGroup = AnnotationGroup(region, get_colon_term("proximal colon"))
             transverseGroup = AnnotationGroup(region, get_colon_term("transverse colon"))
             distalGroup = AnnotationGroup(region, get_colon_term("distal colon"))
-            annotationGroupsAlong += [proximalGroup, transverseGroup, distalGroup]
-            annotationGroupNamesAlong = [['colon', 'proximal colon'],
-                                         ['colon', 'transverse colon'],
-                                         ['colon', 'distal colon']]
```

Comment:
Are the names along/around/through wall still needed? Remove if not.

Reply:
The names are no longer needed, which is why they have been deleted.